test/parallel/test-http-outgoing-finish-writable.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+const common = require('../common');
+const assert = require('assert');
+const http = require('http');
+
+const server = http.createServer(common.mustCall(function(req, res) {
+  assert(res.writable, 'Res should be writable when it is received \
+                          and opened.');
+  assert(!res.finished, 'Res shouldn\'t be finished when it is received \
+                          and opened.');
```

Comment:
Can you use `assert.strictEqual` without a message here and in the rest of the tests?